csrankings-m.csv

```diff
@@ -1482,6 +1482,7 @@ Meng Jin 0002,Shanghai Jiao Tong University,https://yume-sjtu.github.io/,CN2l9GE
 Meng Tang 0001,Univ. of California - Merced,http://www.mengtang.org,6DmW9v0AAAAJ
 Meng Wang 0002,University of Bristol,https://mengwangoxf.github.io,NwDbSggAAAAJ
 Meng Xu 0001,University of Waterloo,https://cs.uwaterloo.ca/~m285xu,NOSCHOLARPAGE
+Meng Yan,Chongqing University,https://yanmeng.github.io,dtxIGDUAAAAJ

```

Comment:
This entry is included twice but it is in fact a disambiguation entry and needs to be removed. See https://dblp.org/pid/131/4289.html.